server/src/testFixtures/java/org/elasticsearch/test/IntegTestCase.java

```diff
@@ -1518,6 +1520,10 @@ public Sessions sqlOperations() {
             });
     }
 
+    public String schema() {
+        return schema;
+    }
+
```

Comment:
Can't we use sqlExecutor.getCurrentSchema() where it's needed?

Reply:
didn't think of that, thx!